components/lists/lists.css

```diff
@@ -0,0 +1,68 @@
+@import "/common.css";
+
+.list-container {
+  display: flex;
+  flex-direction: column;
+  margin: 7rem;
+}
+
+.list-title {
+  font-size: 3rem;
+  font-weight: 600;
+}
+
+.list-content {
+  font-size: 2rem;
+  padding: 1rem;
+}
+
+.ul-section {
+  padding: 2rem;
+}
+
+.section-header {
+  font-size: 2rem;
+}
```

Comment:
maybe you name name this as list-header instead of section-header. Its a suggestion tough.